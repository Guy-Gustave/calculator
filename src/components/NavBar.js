import React from 'react';
import { Link } from 'react-router-dom';
import '../index.css';
/* eslint-disable */

const NavBar = () => {
  return (
    <div className="navbar">
    <h1>Math Magicians</h1>
    <div className="nav">
      <Link to="/">Home </Link>
      <Link to="/App">Calculator </Link>
      <Link to="/Quote">Quote </Link>
    </div>
  </div>
  
  
  );
};

export default NavBar;
