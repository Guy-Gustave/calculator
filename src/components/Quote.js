import React from 'react';
import NavBar from './NavBar';

const Quote = () => {
  <div>
    <NavBar />
    <h1 className="quote">
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding. â€“William Paul Thurston
    </h1>
  </div>;
};

export default Quote;
