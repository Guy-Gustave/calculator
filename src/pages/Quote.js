/* eslint-disable */
import React from 'react';
import NavBar from '../components/NavBar';

const Quote = () => {
  return(
  <div className="quote">
    <NavBar />
    <h3>
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding. â€“William Paul Thurston
    </h3>
  </div>
  );
};

export default Quote;
