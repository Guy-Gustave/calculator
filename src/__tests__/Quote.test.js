import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import NavBar from '../components/NavBar';
import Quote from '../pages/Quote';
import renderer from 'react-test-renderer'
// const Quote = require('../pages/Quote');

it('renders correctly', () => {
  const tree = renderer
    .create(
      <div className="quote">
        <BrowserRouter>
          <NavBar />
        </BrowserRouter>
        <h3>
          Mathematics is not about numbers, equations, computations,
          or algorithms: it is about understanding. â€“William Paul Thurston
        </h3>
      </div>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});