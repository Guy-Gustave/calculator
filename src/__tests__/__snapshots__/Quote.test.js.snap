// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quote"
>
  <div
    className="navbar"
  >
    <h1>
      Math Magicians
    </h1>
    <div
      className="nav"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home 
      </a>
      <a
        href="/App"
        onClick={[Function]}
      >
        Calculator 
      </a>
      <a
        href="/Quote"
        onClick={[Function]}
      >
        Quote 
      </a>
    </div>
  </div>
  <h3>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </h3>
  ,
</div>
`;
